plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestAggregationPushdown.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.pinot;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.pinot.client.PinotClient;
+import io.trino.plugin.pinot.query.AggregationExpression;
+import io.trino.plugin.pinot.query.DynamicTable;
+import io.trino.spi.connector.AggregateFunction;
+import io.trino.spi.connector.AggregationApplicationResult;
+import io.trino.spi.connector.ColumnHandle;
+import io.trino.spi.expression.Variable;
+import io.trino.spi.predicate.TupleDomain;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.OptionalLong;
+
+import static io.airlift.concurrent.Threads.threadsNamed;
+import static io.trino.plugin.pinot.PinotQueryRunner.createSession;
+import static io.trino.plugin.pinot.TestPinotQueryBase.getTestingMetadata;
+import static io.trino.spi.type.BigintType.BIGINT;
+import static io.trino.spi.type.DoubleType.DOUBLE;
+import static java.util.concurrent.Executors.newCachedThreadPool;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class TestAggregationPushdown
```

Comment:
Maybe call it `TestPinotMetadata` since this looks like a unit test of that class